db_mock_impl.go

```diff
@@ -0,0 +1,728 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	_ "github.com/lib/pq"
```

Comment:
When I get away from SQLBoiler I tend to use [`pgx`](https://github.com/jackc/pgx).

Reply:
No preference here this sample code was helped with AI